docker_client.go

```diff
@@ -97,7 +97,7 @@ func splitDockerImage(img string) (string, string, string) {
 func getContainers(client *docker.Client) ([]*RuntimeContainer, error) {
 
 	apiContainers, err := client.ListContainers(docker.ListContainersOptions{
-		All:  false,
+		All:  true,
```

Comment:
Wouldn't it make sense to let the server do the filtering and make the `All` flag dependent on the value of `includeStopped`?